src/components/panel/panel.tsx

```diff
@@ -180,24 +180,24 @@ export class Panel implements ConditionalSlotComponent, InteractiveComponent {
   // --------------------------------------------------------------------------
 
   /**
-   * Emitted when the close button has been clicked.
+   * Emits when the close button has been clicked.
    */
   @Event() calcitePanelDismiss: EventEmitter;
 
   /**
-   * Emitted when the close button has been clicked.
+   * Emits when the close button has been clicked.
    *
    * @deprecated use calcitePanelDismiss instead.
    */
   @Event() calcitePanelDismissedChange: EventEmitter;
 
   /**
-   * Emitted when the content has been scrolled.
+   * Emits when the content has been scrolled.
    */
   @Event() calcitePanelScroll: EventEmitter;
 
   /**
-   * Emitted when the back button has been clicked.
+   * Emits when the back button has been clicked.
```

Comment:
I'm fine with Emits but just wanted to point out MDN.

> The click event is raised when the user clicks on an element. It fires after the [mousedown](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousedown_event) and [mouseup](https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseup_event) events, in that order.

> change events fire when the user commits a value change to a form control. This may be done, for example, by clicking outside of the control or by using the Tab key to switch to a different control.

Seems like they use "is raised" or "fire/s"


> The keydown event fires when the user presses a keyboard key.

Reply:
That makes sense, going to add some click events language to our consistency docs. Also changed a few other pages in this PR that were related, all to "Fires", it seemed more clear than "Is raised", or "Raises".